lib/elements/elements.dart

```diff
@@ -505,3 +505,49 @@ class _ElementTextProperty implements Property {
   // TODO:
   Stream get onChanged => null;
 }
+
+class TabController {
+  List<TabElement> tabElements;
```

Comment:
The variable can be `final` - you can't assign a new list to it - but you can still modify the referenced list itself.